assets/src/dashboard/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import App from './app';
+import './style.css'; // This way the general dashboard styles are loaded before all the component styles.
+
+/**
+ * Initializes the dashboard screen.
+ */
+const initialize = () => {
+  const appElement = document.getElementById('web-stories-dashboard');
```

Comment:
Should we check that this element exists?

Reply:
Hmm if it doesn't exist, something is seriously broken. I don't think we should paper over that.

Reply:
Fair enough.